changelog/12866.doc.rst

```diff
@@ -0,0 +1 @@
+Improved cross-references concerning the :func:`recwarn <_pytest.recwarn.recwarn>` fixture.
```

Comment:
```suggestion
Improved cross-references concerning the :fixture:`recwarn` fixture.
```

Reply:
aha great, so there is a sphinx role for fixtures, nice